packages/common/rollup-configs.js

```diff
@@ -113,9 +84,8 @@ export const tsLibConfig = (pkg, inputFile, format = 'esm') => {
         include: ['src/**/*', '../common/src/**/*'],
         noEmitOnError: true,
       }),
-      ...(format === 'cjs' ? [replaceLodashEsRequire('index.js')] : []),
       writeJSONFile({
-        fileName: 'build-meta.json',
+        fileName: 'build-metadata.json',
```

Comment:
@vojtechszocs can we move the filename to the top of the file in a constant so it's easier to find and change in the future?

Reply:
Chatted with @florkbr and since we're not planning to change this string in future, we can keep it here.